src/test/java/com/blade/kit/CollectionKitTest.java

```diff
@@ -0,0 +1,78 @@
+package com.blade.kit;
+
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.HashMap;
+import org.junit.Assert;
```

Comment:
Provided this repo doesn't have any style requirement that prohibit this, I think it's preferably to import `org.junit.Assert.*;` and collapse your assertions down to, for example: `assertTrue`

Reply:
You could still use import static to get rid of Assert.